specs/pvs/corla.pvs

```diff
@@ -1281,25 +1281,11 @@ BEGIN
 %%% We list various other terms relating to audits here. They will be
 %%% defined and refined in later versions of the domain model.
   contest_margin: TYPE = nat
-  margin: TYPE = real
+  margin: TYPE = nonneg_real
 
   digital_ballot_adjudication: TYPE
   manual_ballot_adjudication: TYPE
 
-% @trace struck from RDPR-6-Jul-2017 old Section 25.1.5
-
-% @description The smallest reported margin in votes between the
-% reported contest winner with the least votes in the contest, and the
-% reported contest loser with the most votes in the contest, divided
-% by the number of ballots counted in that contest.
-
-% @example For example, if the voting system tabulated 10,000 ballots
-% in an audited contest, and the reported winning candidate with the
-% fewest votes received 4,000 votes, and the reported losing candidate
-% with the most votes received 3,500 votes, the diluted margin of the
-% contest is 5% [(4,000 – 3,500) / 10,000]
-  diluted_margin: TYPE FROM margin
-
 % @trace struck from RDPR-6-Jul-2017 old Section 25.1.6
```

Comment:
Recall that traces are going to turn into margin comments with hyperlinks to other artifacts.  So I'm going to leave it in there for now.